.github/workflows/_python-tests.yml

```diff
@@ -50,7 +49,7 @@ jobs:
         shell: bash
         run: pip install --no-index --find-links target/wheels/ chromadb
       - name: Test
-        run: python -m pytest ${{ matrix.test-globs }} ${{ matrix.parallelized && '-n auto --dist worksteal' || '' }} -v --color=yes --durations 10
+        run: python -m pytest ${{ matrix.test-glob }} -n auto --dist worksteal -v --color=yes --durations 10
```

Comment:
I tried to remove all the instances where we had a matrix input of size 1 since it was pointless.